translations/messages.fr.xlf

```diff
@@ -0,0 +1,3063 @@
+                &lt;ol type=&quot;A&quot;&gt;
+                    &lt;li&gt;Choisir la/les capture(s) crée(s) par le dernier mouvement de votre adversaire.&lt;/li&gt;
+                    &lt;li&gt;Faire votre poussée.&lt;/li&gt;
+                    &lt;li&gt;Choisir la/les ligne(s) à capturer que vous venez de créer (en cliquant dessus).&lt;/li&gt;
+                &lt;/ol&gt;
+            &lt;/li&gt;
+        &lt;/ol&gt;&lt;br/&gt;
+        Vous jouez Foncé, une capture est faisable, faites la.</target>
+      </trans-unit>
+      <trans-unit id="8768850104658663274" datatype="html">
+        <source>Congratulations, you have gotten 4 of your pieces back. This is not the most useful capture.
+        Let&apos;s now see how captures can really be useful.</source>
+        <target>Bravo, vous avez récupéré 4 de vos pièces, mais ce n&apos;est pas la capture la plus utile.
+        Voyons maintenant la vraie utilité d&apos;une capture.</target>
+      </trans-unit>
+      <trans-unit id="5296311699765609481" datatype="html">
+        <source>Capturing (2/3)</source>
+        <target>Capture (2/3)</target>
+      </trans-unit>
+      <trans-unit id="6012182624396746560" datatype="html">
+        <source>Here, there are three different ways of capturing.
+        One way does not capture any of your opponent&apos;s pieces.
+        Another way captures one of your opponent&apos;s pieces.&lt;br/&gt;&lt;br/&gt;
+        The last way captures two of your opponent&apos;s pieces, pick that one.</source>
+        <target>Ici, il est possible de capturer de trois façons différentes.
+        L&apos;une ne permet aucune capture de pièce adverse.
+        L&apos;autre permet une capture de pièces adverses.&lt;br/&gt;&lt;br/&gt;
+        La dernière en permet deux, choisissez cette dernière.</target>
+      </trans-unit>
+      <trans-unit id="5909970360139332678" datatype="html">
+        <source>Congratulations, you have gotten back 4 of your pieces and captured 2 of the opponent&apos;s ones.
+        The most you can get is 3 per capture.</source>
+        <target>Bravo, vous avez récupéré 4 de vos pièces et capturé 2 pièces de l&apos;adversaire.
+        Le maximum possible étant 3 par captures.</target>
+      </trans-unit>
+      <trans-unit id="3200525134996933550" datatype="html">
+        <source>Failed, the best capture was taking 2 of your opponent&apos;s pieces</source>
+        <target>Raté, la capture optimale capture 2 pièces adverses.</target>
+      </trans-unit>
+      <trans-unit id="6859741057618390920" datatype="html">
+        <source>Capturing (3/3)</source>
+        <target>Capture (3/3)</target>
+      </trans-unit>
+      <trans-unit id="1697647848066799342" datatype="html">
+        <source>Here, you must capture at the beginning of your turn.
+        This is due to a move of your opponent at the previous turn
+        (even though this is a fictional game for pedagogical purpose).
```

Comment:
fictionnal board for pedagogic purpose ?

Reply:
hmm:
  - fictional : un seul n
  - board: oui ! (changé)
  - pedagogic: pedagogical est plus utilisé

Reply:
Very well, l'essentiel était "board".
Sinon google qui dit ça haha : "a fictitious set for educational purposes."
fiktissiousse set.

Reply:
Mais.... tu n'as pas changé game !